src/cljs/athens/views/pages/settings.cljs

```diff
@@ -78,11 +78,24 @@
   (js/Number (js/localStorage.getItem "debounce-save-time")))
 
 
+(defn init-user
+  []
+  (or (js/localStorage.getItem "user/name"
+       "Socrates")))
+
+
+;(re-frame.core/reg-sub
+;  :user/current
+;  (fn [db _]
+;    (:user db)))
+
+
 (defn init-state
   []
-  {:email         (init-email)
-   :monitoring    (init-monitoring)
-   :backup-time (init-backup-time)})
+  {:email       (init-email)
+   :monitoring  (init-monitoring)
+   :backup-time (init-backup-time)
+   :user        (init-user)})
```

Comment:
TODO: `init-state` should now be part of `:boot/desktop` local-storage initial app-db value

Reply:
I think these four values all bypass re-frame. They should instead use re-frame events to update app-db `:athens/persist` key. re-frame subs also need to be written.